person.rb

```diff
@@ -5,14 +5,19 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name: 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
   end
```

Comment:
- [ ] Please kindly update your add_rental method to properly achieve the many to many relationship
- code snippet
```ruby
def add_rental(book, date)
   Rental.new(date, book, self)
end
```